File: Solutions/typescript/58-length-of-last-word.ts
function lengthOfLastWord(s: string): number {
    // We need to clip the white space
    const trimedS:string = s.trim()
    console.log(trimedS)

    // We need to convert sentence to array of words
    const arrayOFS: string[] = trimedS.split(" ")
    console.log(arrayOFS)

    // Get length of last worf
    const LAST_WORD_INDEX:number = arrayOFS.length-1
    const lastWordLength : number= arrayOFS[LAST_WORD_INDEX].length

    return lastWordLength
};

File: Solutions/C++/58. Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = s.size() - 1;
        int LastWordLength  = 0;
        for(i = i; i > -1; i--){
            if(s[i] == ' '){
                if(LastWordLength == 0)
                    continue;
                else
                    break;
            }
            else
                LastWordLength++;
        }
        return LastWordLength;
    }
};
